servant-client/test/Servant/StreamSpec.hs

```diff
@@ -77,14 +75,24 @@ bob :: Person
 bob = Person "Bob" 25
 
 server :: Application
-server = serve sapi (
-       (return (StreamGenerator (\f r -> f alice >> r bob >> r alice))
-       :: Handler (StreamGenerator Person))
-       :<|>
-       (return (StreamGenerator (\f r -> f alice >> r bob >> r alice))
-       :: Handler (StreamGenerator Person))
-       :<|>
-       emptyServer)
+server = serve sapi
+  $    return (StreamGenerator (\f r -> f alice >> r bob >> r alice))
+  :<|> return (StreamGenerator (\f r -> f alice >> r bob >> r alice))
+  :<|> return (StreamGenerator lotsGenerator)
+  where
+    lotsGenerator f r = do
+      f ""
+      withFile "/dev/urandom" ReadMode $
```

Comment:
can we use `random` / `tf-random` etc. and not drain the (somewhat slow) secure random device?

Reply:
Tried `random` and `bytestring` random, but they used a lot more CPU and failed (without clear error messages) on CI. 

I can remove this test entirely, though